recipes/taglib/all/conandata.yml

```diff
@@ -1,4 +1,10 @@
 sources:
+  "2.0-beta":
```

Comment:
Do you know if this is expected to be released soon? Maybe we can wait for the official 2.0 release?

Reply:
https://mail.kde.org/pipermail/taglib-devel/2023-December/003074.html

> We plan to release version 2.0 in a month.

I will drop the beta version from config.yml and conandata.yml for now and keep the recipe changes for 2.0 intact.